### DISCUSSION TITLE
[Feature Request] Automatically override tailwind theme / styles

### DISCUSSION THREAD

**Discussion by zsolt-dev** [heroui-inc/heroui#4671] [Feature requests]
### Is your feature request related to a problem? Please describe.

NextUI and tailwind themes are inconsistent.

Just one example:
- In tailwind, `rounded-lg` is set to 0.5rem
- nextUI uses `var(--nextui-radius-large);` which is set to `14px`

is there a way to set just one theme in nextUI that will override tailwind theme too? Or what is the correct way to "sync" themes?

### Describe the solution you'd like

Provide the ability to sync themes

### Describe alternatives you've considered

manually doing all the settings...

**Comment by enaluz** [comment]
This is a huge sticking point for me as well. NextUI doesn't play nice with Tailwind once you start overwriting the Tailwind theme to what your design system dictates.
